src/PrestaShopBundle/Routing/Converter/LegacyUrlConverter.php

```diff
@@ -127,30 +117,28 @@ public function convertByUrl($url)
     public function convertByRequest(Request $request)
     {
         $this->router->getContext()->fromRequest($request);
+        $parameters = array_merge($request->query->all(), $request->request->all());
```

Comment:
Isn't `$request->query->all()` already contained in `$request->request->all()`?

Reply:
I don't think so, query is for $_GET parameters, and request for $_POST parameters

Reply:
You're totally right, my mistake.